src/tofromdatavalues.jl

```diff
@@ -1,24 +1,21 @@
 # utilities for converting Row iterators to and from DataValue-based NamedTuple iterators
 # the DataValue-specific definitions are in the Tables.jl __init__ DataValues @require block
-nondatavaluetype(::Type{T}) where {T} = T
-nondatavaluetype(::Type{Union{}}) = Union{}
-datavaluetype(::Type{T}) where {T} = T
-datavaluetype(::Type{Union{}}) = Union{}
-
-Base.@pure function nondatavaluetype(::Type{NT}) where {NT <: NamedTuple{names}} where {names}
-    TT = Tuple{Any[ nondatavaluetype(fieldtype(NT, i)) for i = 1:fieldcount(NT) ]...}
+DataAPI.nondatavaluetype(::Type{T}) where {T} = T
```

Comment:
I know that this function is documented to be owned by Tables.jl, but given how simple this fallback definition is, why not put it in DataAPI instead? That would be simpler for everybody.

Reply:
That's fair.